src/Options/Options.hpp

```diff
@@ -26,6 +26,8 @@ class Node;
 }  // namespace YAML
 /// \endcond
 
+/// Utilities for parsing input files.
+namespace Options {
```

Comment:
How about making this namespace lower-case (`options`)? Not sure how annoying the regex-replace would be to make this change though.. what do you think?